inc/audiences/namespace.php

```diff
@@ -325,29 +324,51 @@ function get_estimate( array $audience ) : ?array {
 			'bool' => [
 				'filter' => [
 					// Set current site.
-					[ 'term' => [ 'attributes.blogId.keyword' => get_current_blog_id() ] ],
+					[
+						'term' => [
+							'attributes.blogId.keyword' => get_current_blog_id(),
+						],
+					],
+
 					// Last 7 days.
-					[ 'range' => [ 'event_timestamp' => [ 'gte' => milliseconds() - ( 7 * 24 * 60 * 60 * 1000 ) ] ] ],
+					[
+						'range' => [
+							'event_timestamp' => [
+								'gte' => Utils\milliseconds() - ( WEEK_IN_SECONDS * 1000 ),
+							],
+						],
+					],
+
 					// Limit event type to pageView.
-					[ 'term' => [ 'event_type.keyword' => 'pageView' ] ],
+					[
+						'term' => [
+							'event_type.keyword' => 'pageView',
+						],
+					],
 				],
 			],
 		],
 		'aggs' => [
-			'estimate' => [ 'cardinality' => [ 'field' => 'endpoint.Id.keyword' ] ],
+			'estimate' => [
+				'cardinality' => [
+					'field' => 'endpoint.Id.keyword',
+				],
+			],
 			'histogram' => [
 				'histogram' => [
 					'field' => 'event_timestamp',
-					'interval' => 6 * 60 * 60 * 1000, // 6 hour chunks.
+					'interval' => 6 * HOUR_IN_SECONDS * 1000, // 6 hour chunks.
 					'extended_bounds' => [
-						'min' => milliseconds() - ( 7 * 24 * 60 * 60 * 1000 ),
-						'max' => milliseconds(),
+						'min' => Utils\milliseconds() - ( WEEK_IN_SECONDS * 1000 ),
+						'max' => Utils\milliseconds(),
```

Comment:
Do these mean the below cache will never be hit?

Reply:
Good question. Probably, yes, but haven't changed from the original here, just refactoring. Best to leave that feedback on #36.